src/win32/win_agent.c

```diff
@@ -173,6 +179,22 @@ int local_start()
     }
     minfo("Using notify time: %d and max time to reconnect: %d", agt->notify_time, agt->max_time_reconnect_try);
 
+    /* Check if is valid server addres */
+    while(i < agt->rip_id){
+
+        if ( strcmp(agt->server[i].rip, val_default) != 0
```

Comment:
The expected behavior in Linux and Windows should be the same. Please don't start the agent if the value is `MANAGER_IP`.